js/default-configs/layouts-config.js

```diff
@@ -1,6 +1,7 @@
 window.flListLayoutConfig = {
   'small-card': {
     'filtersEnabled': true,
+    'chatEnabled': false,
```

Comment:
Why is this `false`? I thought the Directory layout should support chat?

```suggestion
    'chatEnabled': true,
```